cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -83,7 +89,7 @@ def _provisioner(template, provisioning_data, delayed=None):
         fill(provisioning_form, provisioning_data, action=provisioning_form.submit_button)
         flash.assert_no_errors()
 
-        request.addfinalizer(lambda: cleanup_vm(vm_name, provider))
+        request.addfinalizer(lambda: remove_vm(vm_name, provider))
```

Comment:
@bargulg I had a talk with istein. this is an issue in 5.7 and rhevm-s. it's not related to vm cleanup. cleanup runs successfully on 5.6.4.0. this PR is not needed I guess